### ISSUE TITLE
alias encryption key issues.

### ISSUE THREAD

**Issue by benp1000** [syscoin/blockmarket-desktop-public#358]
2 out of 3 alias had "missing encryption key" warnings- I never had the option to use the generate button like others reported having- so I tried to update them individually manually. I found on the "edit alias" form I couldn't select the right address for the alias (address not showing as option) without first switching alias view from advanced to basic and back. After that the address would show in the drop down list - but no privkey or pubkey data would be auto filled out. So I had to select a different address from the drop down menu and then the info would fill out and then I switched back to my aliases address and the pub key privkey info would update in the fields for that address. I was then able to put through the transaction to update the alias- but it still continued to show (after many confirmations) the original missing encryption key warning and additionally it gives me a new warning, which hadn't occurred before the update transaction, when opening the alias form. "Error looking up private key. Unable to locate privatekey for (alias name)"
I tried reassigning my alias to a different address and that worked (the reassignment worked) but it brought over to the new address the same aforementioned issues.
However creating a new address and assigning a new alias to it worked fine with none of these issues.

**Comment by benp1000** [comment]
seem to have them mostly sorted now. they all say "key cypher invalid" though in the alias form privkey field". And the alias that I assigned to a different address says "Data proof mismatch: The data related to this identity may have been tampered with" - I also can't reassign it back to my old address. transaction seemed to work but it doesn't change back.

 One thing to note- I couldn't update my aliases unless the alias that was being updated had enough funds in it. I thought it was suppose to work as long as there were enough funds in the alias that according to the account menu I was working from.

**Comment by benp1000** [comment]
"key cypher invalid" issue has also resolved on all aliases except for reassigned address alias

**Comment by benp1000** [comment]
its still very buggy with the alias form. Have to switch to advanced/basic view back and forth in order for the selected aliases info to come up once you've selected it.

**Comment by benp1000** [comment]
got everything sorted now. except offchain data issue with one alias. creating separate GitHub issue for that

**Comment by Forlan84** [comment]
I confirm that I have the same issue as in pic3:
My alias doesn't show the "missing encryption key" warning does however have a warning once you open the form on the privkey field saying "key cypher invalid".

**Comment by q-src** [comment]
@Forlan84 Maybe this is only due to your wallet being locked. 
In this case, I can confirm the behaviour. However, as soon as the wallet gets unlocked and the "Edit Alias" form reopened, the red warning sign turns into a green check mark which says "Key Cipher Valid".
Once the wallet is locked again, the warning appears again. So it seems like the wallet needs to be unlocked in order to validate the key cipher. Can you confirm this @syscoin?

**Comment by Forlan84** [comment]
@q-src No! I get a red mark regardless of the wallet state (locked/unlocked).
